test/Examples_Tests/src/Signatures_Spec.enso

```diff
@@ -8,7 +8,7 @@ add_specs suite_builder = suite_builder.group "Standard.Base Signature Checks" g
     setup = Api_Setup.create 'Standard' 'Base' [ "--docs=api" ]
 
     group_builder.specify "Standard.Base.Meta" <|
-        snapshot = setup.api_dir / "Standard" / "Base" / "Meta.md"
+        snapshot = setup.api_dir / "Meta.md"
```

Comment:
Skipping `namespace` and `name` from the directory structure? That is probably good idea. CCing @radeusgd

Reply:
The idea is that the directory hierarchy under `docs/api` should follow the one in `src`. It is tested in https://github.com/enso-org/enso/blob/30904f6d9d619611cb7353822ef8c03796dc968b/engine/runtime-integration-tests/src/test/java/org/enso/compiler/dump/test/DocsGenerateTest.java#L282-L312